src/main.rs

```diff
@@ -666,6 +670,23 @@ fn get_total_loc(languages: &tokei::Languages) -> usize {
         .fold(0, |sum, val| sum + val.code)
 }
 
+fn get_creation_time() -> Option<String> {
+    let output = Command::new("git")
+        .arg("log")
+        .arg("--reverse")
+        .arg("--pretty=oneline")
+        .arg("--format=\"%ar\"")
+        .output()
+        .expect("Failed to execute git.");
+
+    let output = String::from_utf8_lossy(&output.stdout);
+
+    match output.lines().next() {
+        Some(val) => Some(val.to_string().replace('"', "")),
+        None => None
```

Comment:
We might not even need to handle `None` since the program will error and exit before this in case there are no commits